components/handlers/module_edit_prerequisites.py

```diff
@@ -0,0 +1,45 @@
+'''
+    This module contains the handler for web requests pertaining to
+    the editing of a module's prerequisites.
+'''
+
+
+import json
+from app import RENDER
+import web
+from components import model, session
+
+
+class EditModulePrerequisites(object):
+    '''
+        This class handles the editing of a module's prerequisites.
+    '''
+    def GET(self):
+        '''
+            Handles the loading of the 'Edit Module Prerequisites' page.
+        '''
+        if not session.validate_session():
+            raise web.seeother('/login')
+        else:
+            input_data = model.validate_input(web.input(), ["code"])
+            module_code = input_data.code
+
+            prerequisites = model.get_prerequisite_units(module_code)
+            return RENDER.moduleEditPrerequisite(module_code, prerequisites)
+
+
+    def POST(self):
+        '''
+            Handles the submission of updated module prerequisites
+            for a target module.
+        '''
+        isSucessfullyUpdated = False
+
+        input_data = model.validate_input(web.input(), ["code"], show_404=False)
+
+        if input_data:
+            module_code = input_data.code
+            prerequisites = json.loads(input_data.prerequisites)
+            isSucessfullyUpdated = model.edit_prerequisite(module_code, prerequisites)
```

Comment:
Edit prereq and preclusions are still not accepting lowercase module codes. Should convert module_code to .upper() before passing it into model.edit_prerequisite